### PR TITLE
All blocks page wire up data with pagination support

### LINKED ISSUE #188: All blocks page: wire up data with pagination support

### PR DESCRIPTION
**Pull request by prin-r**
Fixes #188

### CODE REVIEW

File: scan/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "js-sha256": "^0.9.0",
     "moment": "^2.24.0",
     "react": "^16.8.0-0",
+    "react-highlight": "^0.12.0",
```
Comment (sorawit) [inline scan/package.json]: Remove

File: scan/src/bindings/ReactHighlight.re [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[@bs.obj] external makeProps: (~children: React.element, unit) => _ = "";
+
+[@bs.module "react-highlight"]
```
Comment (sorawit) [inline scan/src/bindings/ReactHighlight.re]: also here

File: scan/src/images/Images.re [hunk 3]
```diff
@@ -5,6 +5,7 @@
 [@bs.module] external rightArrow: string = "./right-arrow.svg";
 [@bs.module] external telegramLogo: string = "./telegram-logo.svg";
 [@bs.module] external twitterLogo: string = "./twitter-logo.svg";
+[@bs.module] external textDocument: string = "./text-document.svg";
```
Comment (sorawit) [inline scan/src/images/Images.re]: also here? and below?

File: scan/src/pages/BlockHomePage.re [hunk 4]
```diff
@@ -14,73 +14,83 @@ module Styles = {
...
 };
 
-let renderBody = (idx, (height, timestamp, proposer, totalTx, totalFee, blockReward)) => {
+let renderBody = (idx, block: BlockHook.Block.t) => {
+  let height = block.height;
+  let timestamp = block.timestamp;
+  let proposer = block.proposer->Address.toOperatorBech32;
+  let totalTx = block.numTxs;
+
   <TBody key={idx |> string_of_int}>
```
Comment (sorawit) [inline scan/src/pages/BlockHomePage.re]: I think we can just use block height as index @evilpeach ?

File: scan/src/pages/ScriptIndexPage.re [hunk 5]
```diff
@@ -128,9 +139,64 @@ let make = (~codeHash, ~hashtag: Route.script_tab_t) => {
            <VSpacing size=Spacing.lg />
            <TxsTable />
            <VSpacing size=Spacing.lg />
-           <LoadMore />
+           <LoadMore onClick={_ => ()} />
+         </div>
```
Comment (sorawit) [inline scan/src/pages/ScriptIndexPage.re]: This is not relevant..

**Review by sorawit — CHANGES_REQUESTED** [review]
@evilpeach Please review

File: scan/src/pages/BlockHomePage.re (hunk 4 above)
Comment (evilpeach) [reply scan/src/pages/BlockHomePage.re]: Sure

File: scan/package.json (hunk 1 above)
Comment (evilpeach) [reply scan/package.json]: I think we should complete #214 first then rebase it to this PR

**Review by sorawit — APPROVED** [review]
Good to me. Wait for @evilpeach

**Pull request merged**